R/plot.R

```diff
@@ -2,175 +2,298 @@
+#'
+#' @param ref Reference treatment.
+#' @return Text as above.
 make_refText = function(ref) {
-  y <- paste("All outcomes are versus the reference treatment:", ref)
-  return(y)
+  return(paste("All outcomes are versus the reference treatment:", ref))
 }
 
-# 2b Treatment comparison and rank table
+
+
+#' 2b Treatment comparison and rank table
+#' 
+#' @param freq List of NMA results created by freq_wrap().
+#' @param modelranfix "fixed" or "random".
+#' @param rankopts "good" or "bad", referring to smaller outcome values.
+#' @return Ranking table created by netmeta::netleague().
 make_netrank <- function(freq, modelranfix, rankopts) {
-  league <- netleague(freq$net1, 
-                      comb.random=(modelranfix=="random"), comb.fixed = (modelranfix=="fixed"), 
-                      digits =2, seq= netrank(freq$net1, small = rankopts))
-  if (modelranfix=="random"){
-    leaguedf<- as.data.frame(league$random)
-  }
-  else {
-    leaguedf<- as.data.frame(league$fixed)
+  league <- netmeta::netleague(freq$net1, comb.random = (modelranfix == "random"),
+                               comb.fixed = (modelranfix == "fixed"), digits = 2,
+                               seq = netmeta::netrank(freq$net1, small = rankopts)
+                               )
+  if (modelranfix == "random") {
+    leaguedf <- as.data.frame(league$random)
+  } else if (modelranfix == "random") {
+    leaguedf <- as.data.frame(league$fixed)
+  } else {
+    stop("modelranfix must be 'fixed' or 'random'") 
   }
   return(leaguedf)
 }
 
-# 2c Inconsistency
 
+
+#' 2c Inconsistency
+#'
+#' @param freq List of NMA results created by freq_wrap().
+#' @param modelranfix "fixed" or "random".
+#' @return Data frame of inconsistency data created by netsplitresult.df().
 make_Incon <- function(freq, modelranfix) {
-  incona <- netsplit(freq$net1)
+  incona <- netmeta::netsplit(freq$net1)
   return(netsplitresult.df(incona, modelranfix))
 }
 
-# 3a Forest plot 
+
+
+#' 3a Forest plot 
+#' 
+#' @param model Various model output created by baye().
+#' @param metaoutcome "Continuous" or "Binary".
+#' @param bayesmin x-axis limit minimum.
+#' @param bayesmax x-axis limit maximum.
+#' @return Forest plot created by gemtc::forest().
 make_Forest <- function(model, metaoutcome, bayesmin, bayesmax) {
-  if (metaoutcome=="Binary") {
-    return(gemtc::forest(model$mtcRelEffects, digits=3, xlim=c(log(bayesmin), log(bayesmax))))
-  } else if (metaoutcome=="Continuous") {
-    return(gemtc::forest(model$mtcRelEffects, digits=3, xlim=c(bayesmin, bayesmax)))
+  if (metaoutcome == "Binary") {
+    return(gemtc::forest(model$mtcRelEffects, digits = 3, xlim = c(log(bayesmin), log(bayesmax))))
+  } else if (metaoutcome == "Continuous") {
+    return(gemtc::forest(model$mtcRelEffects, digits = 3, xlim = c(bayesmin, bayesmax)))
+  } else {
+    stop("metaoutcome must be 'Continuous' or 'Binary'") 
   }
 }
 
-# 3b Comparison of all treatment pairs
+
+
+#' 3b Comparison of all treatment pairs
```

Comment:
Description should describe what is being done here. Is it creating a table, a data frame, making a calculation?